Packs/Code42/ReleaseNotes/2_0_2.md

```diff
@@ -1,4 +1,5 @@
 
 #### Integrations
 ##### Code42
+- Fix bug where File Category would not map correctly when creating incidents from Code42 alerts.
```

Comment:
IF this needs a separate release let me know, but it seems like we have 6 days until the next release.